.changeset/clever-vans-help.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sl-design-system/menu': patch
+'@sl-design-system/sanoma-learning': patch
+---
+
+Do not transform token parts to kebab case, only add '-' in between
```

Comment:
Maybe you can also add some information that it's added due to contextual tokens usage? :)